ns/nns.go

```diff
@@ -116,50 +79,23 @@ func (n *NNS) Dial(address string) error {
 //
 // See also https://docs.neo.org/docs/en-us/reference/nns.html.
 func (n *NNS) ResolveContainerName(name string) (cid.ID, error) {
-	res, err := n.neoClient.call(n.nnsContract, "resolve", []smartcontract.Parameter{
-		{
-			Type:  smartcontract.StringType,
-			Value: name + ".container",
-		},
-		{
-			Type:  smartcontract.IntegerType,
-			Value: big.NewInt(int64(nns.TXT)),
-		},
-	})
-	if err != nil {
-		return cid.ID{}, fmt.Errorf("invoke NNS contract: %w", err)
-	}
-
-	if res.State != vmstate.Halt.String() {
-		return cid.ID{}, fmt.Errorf("NNS contract fault exception: %s", res.FaultException)
-	} else if len(res.Stack) == 0 {
-		return cid.ID{}, errors.New("empty stack in invocation result")
-	}
-
-	itemArr, err := res.Stack[len(res.Stack)-1].Convert(stackitem.ArrayT) // top stack element is last in the array
+	arr, err := unwrap.Array(n.invoker.Call(n.nnsContract, "resolve",
+		name+".container", big.NewInt(int64(nns.TXT)),
```

Comment:
I don't think you need `big.NewInt()` wrapping now.

Reply:
fixed, also fixed tests